messaging/kafka-avro-reactive-messaging/src/test/java/io/quarkus/ts/openshift/messaging/kafka/StrimziKafkaResource.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.ts.openshift.messaging.kafka;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import io.strimzi.StrimziKafkaContainer;
+import org.jboss.logging.Logger;
+import org.testcontainers.containers.Network;
+import org.testcontainers.lifecycle.Startables;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class StrimziKafkaResource implements QuarkusTestResourceLifecycleManager {
```

Comment:
I would move these classes to a separate package, so we can directly see what are the tests and what the resources/containers.